test/integration/change-streams/change_streams.prose.test.ts

```diff
@@ -547,130 +380,6 @@ describe('Change Stream prose tests', function () {
       });
     });
 
-    // 12. For a ChangeStream under these conditions:
```

Comment:
suggest: removing these tests from your specifications PR too, if you haven't already

Reply:
Removed from my spec PR.